packages/venia-ui/.storybook/getEnvironmentVariable.js

```diff
@@ -0,0 +1,30 @@
+const envVarDefinitions = require('@magento/pwa-buildpack/envVarDefinitions.json');
+
+/**
+ * Returns the value of the first variable in any section whose name matches the input.
+ * @param {string} targetVariable - The name of the environment variable to find.
+ */
+const getEnvironmentVariable = targetVariable => {
```

Comment:
Can you just use `loadEnvironment` instead of this custom function? `loadEnvironment` should be able to do exactly what this function is doing.

Reply:
Hmmm, actually I'm not sure

I tried to use it but it seems to have a dependency exactly where I need, in the MAGENTO_BACKEND_URL, when I use loadEnvironment to get this variable it throws a exception:

```
  ⓧ  Missing required environment variables:
     MAGENTO_BACKEND_URL: Connect to an instance of Magento 2.3 by specifying its public domain name. (eg. "https://master-7rqtwti-mfwmkrjfqvbjk.us-4.magentosite.cloud/")
```

I tried using this code: 
```
const loadEnvironment = require('@magento/pwa-buildpack/lib/Utilities/loadEnvironment');
loadEnvironment('MAGENTO_BACKEND_URL')
```

Do you know why is this happening? Maybe I'm using it in a wrong way

Reply:
I'm not sure about this either. The function signature of [loadEnvironment](https://github.com/magento/pwa-studio/blob/develop/packages/pwa-buildpack/lib/Utilities/loadEnvironment.js#L128) is

```js
function loadEnvironment(dirOrEnv, logger = prettyLogger) {
```

so it doesn't look like it wants the name of the environment variable.

Reply:
@LucasCalazans check out other places where we use it. For example in `create-custom-origin`:

```js
const projectConfig = loadEnvironment(projectRoot);
        if (projectConfig.error) {
            failExpected(projectConfig.error);
        }
        const config = projectConfig.section('customOrigin');
        // etc
```

You pass it a directory and it returns an object with props on it. I'm not sure how well it is documented unfortunately. It may take some experimenting.